32-bit int overflow in TranslateFile MapSize

### Expected behavior

32-bit compiles

### Actual behavior

```./translate.go:94:10: constant 10737418240 overflows int```

### Steps to reproduce the behavior

`make release-build GOOS=linux GOARCH=386`

### Information about your environment (OS/architecture, CPU, RAM, cluster/solo, configuration, etc.)

Mac OS

--- a/translate.go
+++ b/translate.go
@@ -25,8 +25,6 @@ const (
 )
 
 const (
-	DefaultMapSize = 10 * (1 << 30) // 10GB
-
 	DefaultReplicationRetryInterval = 1 * time.Second
 )
 

--- a/translate_mapsize_386.go
+++ b/translate_mapsize_386.go
@@ -1,0 +1,5 @@
+package pilosa
+
+// DefaultMapSize is the default size of mapped memory for the translate store.
+// It is passed as an int to syscall.Mmap and so must be < 2^31
+const DefaultMapSize = (1 << 31) - 1 // 2GB

--- a/translate_mapsize_all64bitsystems.go
+++ b/translate_mapsize_all64bitsystems.go
@@ -1,0 +1,8 @@
+// +build !386
+
+package pilosa
+
+// DefaultMapSize is the default size of mapped memory for the translate store.
+// It is passed as an int to syscall.Mmap and so can only be larger than 2^31 on
+// 64bit systems.
+const DefaultMapSize = 10 * (1 << 30) // 10GB

